solutions/community-deploy/cloud/AWS/README.md

```diff
@@ -84,44 +88,27 @@ for ip in $PUBLIC_IP;do rsync -avzrP -e 'sudo ssh -i cortx.pem -o StrictHostKeyC
 
 **Note:** Become the **root** user after logged in to the primary node by running `sudo su` command.
 ```
-AWS_PRIMARY_IP=$(cat ip_public.txt | jq '.[0]'| tr -d '",[]')
-ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@$AWS_PRIMARY_IP
-git clone https://github.com/Seagate/cortx-re && cd $PWD/cortx-re/solutions/community-deploy
-time bash -x ./build-cortx.sh
-```
-- Save and compress the cortx build images by running following command,
-```
-echo "Wait till the operation is completed..." && docker save $(docker images | sed '1d' | awk '{print $1 ":" $2 }') -o /tmp/cortximages.tar
-```
-- Execute the following command to copy the cortx build images from **primary node to all the worker nodes using private ip address**,
+export PRIMARY_IP=$(cat ip_public.txt | jq '.[0]'| tr -d '",[]')
```

Comment:
Can avoid two separate instructions for cloning `cortx-re` repo. We can have only one instruction and mention that If you do not provide -b <branch/tag>, then it will use default main branch